rasa/cli/x.py

```diff
@@ -112,10 +112,24 @@ def _overwrite_endpoints_for_local_x(
     from rasa.utils.endpoints import EndpointConfig
     import questionary
 
+    # Checking if endpoint.yml has existing url and wait time values set, if so give warning we are overwriting
+    # the endpoint.yml file.
+    custom_wait_time_pulls = endpoints.model.kwargs.get("wait_time_between_pulls")
+    custom_url = endpoints.model.url
+    default_rasax_model_server_url = (
+        f"{rasa_x_url}/projects/default/models/tag/production"
+    )
+
+    if custom_url is not None and custom_url != default_rasax_model_server_url:
```

Comment:
the 2nd part of the condition also covers the first condition
```suggestion
    if custom_url != default_rasax_model_server_url:
```